src/lib/y2journal/query_presenter.rb

```diff
@@ -96,6 +96,16 @@ def interval_description
       end
     end
 
+    # Split journalctl timestamp at "—"; localize and return the joined string back
+    def self.localize_timestamp(boot)
+      start_time = boot[:timestamps][:first]
+      end_time   = boot[:timestamps][:last]
+
+      # TRANSLATORS: %<first>s and %<last>s will be replaced by already localized
+      # timestamps indicating when the system booted and when it was shut down
+      _(format("%<first>s to %<last>s", first: start_time, last: end_time))
```

Comment:
Shouldn't you apply some format to the timestamps here? Right now it would look like this (in my current Ruby version, that may change in the future):

```
2021-08-27 10:31:01 +0200 to 2021-08-30 15:36:09 +0200
```

I think that looks even uglier than before.

What about something like this?

```ruby
def self.localize_timestamp(boot)
  start_time = Yast::Builtins.strftime(boot[:timestamps][:first], "%c")
  end_time = Yast::Builtins.strftime(boot[:timestamps][:last], "%c")

 # TRANSLATORS: blah
 format(_("%<first>s to %<last>s"), first: start_time, last: end_time)
end
```

Reply:
@ancorgs I'm sorry! There are a couple of changes that I forgot to push. I thought of using the existing `TIME_FORMAT` that is defined in the same file:

```rb
TIME_FORMAT = "%b %d %H:%M:%S".freeze
```

Would this be acceptable? I've also fixed the `format(_(...))` problem that you called out earlier.

Reply:
Sure, it fully makes sense.

Again, sorry for the late reply. :-(

Reply:
Those extra changes were never pushed. But it's a shame to leave this contribution in this almost-finished status. So I guess I will cherry-pick @srinidhibs commits in a new branch and try to fix the standing issues myself.

Unless @srinidhibs still has those commits around and he manages to finally push them, of course. :wink:

Reply:
Hey @ancorgs! I'm sorry for dropping the ball on this one! Can you please give me a couple of days? I'll try to find out if I've unfinished work in my local branch and try to close this soon.